apps/semver-yeasy/semver-yeasy.sh

```
@@ -176,8 +206,8 @@ tag)
     git config --global user.name 'github-actions'
     if [ "${repo_type}" = 'SINGLE_APP' ]; then
         if [ "${SEMVERYEASY_CHANGED}" = 'true' ]; then
-        ${GITVERSION_EXEC_PATH} $(pwd) /config "${CONFIG_FILE}"
-        gitversion_calc=$(${GITVERSION_EXEC_PATH} $(pwd) /config "${CONFIG_FILE}")
+        ${GITVERSION_EXEC_PATH} $(pwd) /nonormalize /config "${CONFIG_FILE}"
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)